desktop/apps/consignments2/components/submission_flow/index.js

```diff
@@ -56,7 +55,7 @@ const mapStateToProps = (state) => {
 }
 
 const mapDispatchToProps = {
-  resizeWindowAction: resizeWindow
+  responsiveWindowAction: resizeWindow
```

Comment:
Can we just get rid of this `mapDispatchToProps ` entirely? Looks like it may have been an accident that I included this method in the first place.